utils/data.ts

```diff
@@ -0,0 +1,16 @@
+export const navLinks = [
```

Comment:
this is the app routing config. should not be in utils

Reply:
emotion  package is for the material ui
navLinks moved to routing file